pkg/api/api.go

```diff
@@ -197,6 +203,10 @@ func Router() *gin.Engine {
 			{
 				login.POST("", auth.Login)
 			}
+			oidc := unAuthorizedv1.Group("/users/oidc")
+			{
+				oidc.POST("", auth.CreateOidcUser)
+			}
```

Comment:
The request to create a user should be validated with a JWKS at least????

Reply:
CreateOidcUser function has validation in it